src/app/components/feed/SaveFeed.js

```diff
@@ -473,10 +570,81 @@ const SaveFeed = (props) => {
                 description="On feed edit page, show the last time the feed was changed. The placeholder 'timeAgo' is something like '10 minutes ago'."
               />
             </div>
-          </div> : null
+          </div>
+          : null
         }
       </div>
 
+      {/* "Delete" dialog */}
+      <ConfirmProceedDialog
+        open={showDeleteDialog}
+        title={
+          feed.saved_search_id ?
+            <FormattedMessage
+              id="saveFeed.deleteSharedFeedWarningTitle"
+              defaultMessage="Are you sure you want to delete this shared feed?"
+              description="'Delete' here is an infinitive verb"
+            />
+            :
+            <FormattedMessage
+              id="saveFeed.deleteSharedFeedTitle"
+              defaultMessage="Delete Shared Feed?"
+              description="'Delete' here is an infinitive verb"
+            />
+        }
+        body={
+          feed.saved_search_id ?
+            <p className={styles.saveFeedDialogDivider}>
+              <FormattedHTMLMessage
+                id="saveFeed.deleteSharedFeedConfirmationDialogWaningBody"
+                defaultMessage="This shared feed is available to all users of <b>{orgName}</b>. After deleting it, no user will be able to access it.<br /><br />"
```

Comment:
Please use `strong` instead of `b` for marking text as important here and the other places in the file. I know the `react-intl` docs actually use `<b>` in their examples for `FormattedHTMLMessage`, [they should not](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b)!